praetorian_cli/ui/aegis/tmux.py

```
@@ -0,0 +1,303 @@
+                "split-window", flag, "-P", "-F", "#{pane_id}",
+                command,
+            )
+
+        # Rebalance: main-vertical keeps TUI on the left at ~30%
+        try:
+            self._tmux_cmd("select-layout", "main-vertical")
+            self._tmux_cmd(
+                "set-window-option", "main-pane-width", "30%",
+            )
+        except Exception:
+            pass
```

Ruff (0.15.7)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)